actnow/accounts/serializers.py

```diff
@@ -13,7 +13,11 @@ class Meta:
     def create(self, validated_data):
         if not validated_data["username"]:
             del validated_data["username"]
-        user = User.objects.create_user(**validated_data)
+
+        request_data = self.context["request"].data.copy()
```

Comment:
Won't this include every field? I'm worried about sensitive fields like password being passed on as well.

Reply:
We should delete all user account details before sending them right?

Reply:
Yes (s)sir(K)!

Reply:
This has been updated @kilemensi